eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -3,6 +3,9 @@
 
 <UsageData>
   <IgnorePatterns>
+    <Usage Id="Microsoft.DotNet.XliffTasks" />
+    <UsagePattern IdentityGlob="Microsoft.Build.*" />
```

Comment:
This one is **probably** also unnecessary, and you just need a source-build-reference-packages dependency. https://github.com/dotnet/aspnetcore/blob/main/eng/Version.Details.xml#L337-L341

Reply:
My guess is that this was for Microsoft.Build.Traversal?